manage

```diff
@@ -316,9 +319,27 @@ pyenv.uninstall() {
 }
 
 format.python() {
-    build_msg TEST "[format.python] black \$BLACK_TARGETS"
-    pyenv.cmd black "${BLACK_OPTIONS[@]}" "${BLACK_TARGETS[@]}"
-    dump_return $?
```

Comment:
The `ump_return $?` should not be removed .. add it right below the both `pyenv.cmd black "${BLACK_OPTIONS[@]}" ...` commands